scss-test.scss

```diff
@@ -0,0 +1,10 @@
+.unsorted {
+  color: #FFFFFF;
+  background: red;
+  .test {
+    color: #aaa
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Declaration should be terminated by a semicolon